### PR TITLE
docs: refactor rest api

### PR DESCRIPTION
**Pull request by dabeeeenster**

### CODE REVIEW

File: docs/docs/clients/rest.md [hunk 1]
```diff
@@ -57,19 +57,19 @@ For SaaS customers, the URL to hit for this API is [`https://edge.api.flagsmith.
...
 administrative front end via the API.
 
-To authenticate, you can use the API Token associated with your account. This can be found in the "Account" page from
-the top navigation panel. You need to provide the token as an HTTP header of the form:
+To authenticate, you can use the API Token associated with your Organisation. This can be found in the `Organisation`
+page from the top navigation panel. You need to create a token and then provide it as an HTTP header:
 
 ```bash
-authorization: Token <API TOKEN FROM ACCOUNT PAGE>
+authorization: Token <API TOKEN FROM ORGANISATION PAGE>
```
Comment (gagantrivedi) [inline docs/docs/clients/rest.md]: for api key this will be
`Authorization:  Api-Key  <api-key>`

**Review by gagantrivedi — CHANGES_REQUESTED** [review]

File: docs/docs/clients/rest.md (hunk 1 above)
Comment (dabeeeenster) [reply docs/docs/clients/rest.md]: OK fixed

File: docs/docs/clients/rest.md [hunk 2]
```diff
@@ -57,19 +57,19 @@ For SaaS customers, the URL to hit for this API is [`https://edge.api.flagsmith.
...
+Authorization: Api-Key <API TOKEN FROM ORGANISATION PAGE>
 ```
 
 For example, to create a new Environment:
 
 ```bash
 curl 'https://api.flagsmith.com/api/v1/environments/' \
     -H 'content-type: application/json' \
-    -H 'authorization: Token <API TOKEN FROM ACCOUNT PAGE>' \
+    -H 'authorization: Token <API TOKEN FROM ORGANISATION PAGE>' \
```
Comment (gagantrivedi) [inline docs/docs/clients/rest.md]: -H 'authorization: Api-Key <API TOKEN FROM ORGANISATION PAGE>'  \

Comment (dabeeeenster) [reply docs/docs/clients/rest.md]: OK fixed

**Review by gagantrivedi — APPROVED** [review]

**Pull request merged**